primitives/session-keys/src/inherent.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019-2022 Moonsong Labs
+// This file is part of Moonkit.
```

Comment:
Again, not part of Moonkit, although we could give them credit. Otherwise, we could just import this from Moonkit, if we want to get updates from them.

Reply:
Its not exactly their pallet, I made some updates but forgot to remove that text. We should give them credit somewhere still as the base of the randomness pallet comes from them